doc/source/conf.py

```diff
@@ -385,7 +397,40 @@ def filter_out_undoc_class_members(member_name, class_name, module_name):
         return ""
 
 
+def get_api_groups(method_names, class_name, module_name):
```

Comment:
nit: `_get_api_group`? seems that they are just going to be assigned to the `FILTERS`?